client/src/components/CardItem/CardItem.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+import {Typography, Box, styled} from "@mui/material";
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+
+const CardItemWrapper = styled("div")({
+    minWidth: "265px",
+    maxWidth: "265px",
+    height: "400px",
+    background: `url('../../assets/images/hotDeals/aerial-view-of-colorful-mixed-forest-shrouded-in-morning-fog-on-a-beautiful-autumn-day.jpg')`,
+    backgroundSize: "cover",
+    color: "white",
+    position: "relative",
+    cursor: "pointer",
+
+    "&::after": {
+        content: "''",
+        position: "absolute",
+        width: "100%",
+        height: "100%",
+        background: "rgba(0,0,0,0.3)",
```

Comment:
```suggestion
        background: "rgba(0,0,0,0.5)",
```

i would suggest making it a little darker

Reply:
done